source/StigData/Processed/VMW-vSphere-7-0-1.2.xml

```diff
@@ -0,0 +1,1352 @@
+      <DuplicateOf />
+      <IsNullOrEmpty>False</IsNullOrEmpty>
+      <LegacyId>
+      </LegacyId>
+      <OrganizationValueRequired>False</OrganizationValueRequired>
+      <OrganizationValueTestString />
+      <RawString>From the vSphere Client, go to Host and Clusters.
+
+Select a vCenter Server &gt;&gt; Configure &gt;&gt; Settings &gt;&gt; Advanced Settings.
+
+Verify the "config.log.level" value is set to "info".
+
+or
+
+From a PowerCLI command prompt while connected to the vCenter server, run the following command:
+
+Get-AdvancedSetting -Entity &lt;vcenter server name&gt; -Name config.log.level and verify it is set to "info".
+
+If the "config.log.level" value is not set to "info" or does not exist, this is a finding.</RawString>
+    </Rule>
+    <Rule id="V-256355" severity="medium" conversionstatus="pass" title="SRG-APP-000516" dscresource="VMHostAdvancedSettings">
+      <AdvancedSettings>'' = '30'</AdvancedSettings>
+      <Description>&lt;VulnDiscussion&gt;By default, vCenter will change the "vpxuser" password automatically every 30 days. Ensure this setting meets site policies. If it does not, configure it to meet password aging policies. 
+
+Note: It is very important the password aging policy is not shorter than the default interval that is set to automatically change the "vpxuser" password to preclude the possibility that vCenter might be locked out of an ESXi host.&lt;/VulnDiscussion&gt;&lt;FalsePositives&gt;&lt;/FalsePositives&gt;&lt;FalseNegatives&gt;&lt;/FalseNegatives&gt;&lt;Documentable&gt;false&lt;/Documentable&gt;&lt;Mitigations&gt;&lt;/Mitigations&gt;&lt;SeverityOverrideGuidance&gt;&lt;/SeverityOverrideGuidance&gt;&lt;PotentialImpacts&gt;&lt;/PotentialImpacts&gt;&lt;ThirdPartyTools&gt;&lt;/ThirdPartyTools&gt;&lt;MitigationControl&gt;&lt;/MitigationControl&gt;&lt;Responsibility&gt;&lt;/Responsibility&gt;&lt;IAControls&gt;&lt;/IAControls&gt;</Description>
+      <DuplicateOf />
+      <IsNullOrEmpty>False</IsNullOrEmpty>
+      <LegacyId>
+      </LegacyId>
+      <OrganizationValueRequired>False</OrganizationValueRequired>
+      <OrganizationValueTestString />
+      <RawString>From the vSphere Client, go to Host and Clusters.
+
+Select a vCenter Server &gt;&gt; Configure &gt;&gt; Settings &gt;&gt; Advanced Settings. 
+
+Verify "VirtualCenter.VimPasswordExpirationInDays" is set to "30".
+
+or
+
+From a PowerCLI command prompt while connected to the vCenter server, run the following command:
+
+Get-AdvancedSetting -Entity &lt;vcenter server name&gt; -Name VirtualCenter.VimPasswordExpirationInDays
+
+If the "VirtualCenter.VimPasswordExpirationInDays" is set to a value other than "30" or does not exist, this is a finding.</RawString>
+    </Rule>
+    <Rule id="V-256356" severity="medium" conversionstatus="pass" title="SRG-APP-000516" dscresource="VMHostAdvancedSettings">
+      <AdvancedSettings>'' = '32'</AdvancedSettings>
```

Comment:
this setting is not correct should be 
vpxd.event.syslog.enabled" = '32'

Reply:
Corrections Made